src/emu/common/include/common/fileutils.h

```diff
@@ -52,6 +52,10 @@ namespace eka2l1::common {
     */
     bool remove(const std::string &path);
 
+    /* !\brief Get the content of a UTF-16 LE encoded file as UTF8 string.
+    */
+    std::string get_file_content_u16(const std::string &path);
```

Comment:
Can you invest in dynamic_ifile? It can dynamically read text file depending on the format (UTF8 or UTF16)

Reply:
Changed to use that class, didn't find that before